sycl/doc/GettingStartedAlveo.md

```diff
@@ -283,16 +290,15 @@ Some checks may fail but that may not be an issue.
 ## Compiling and running an application
 
 The typical environment is setup with something like
+FIXME
 ```bash
-export XILINX_SDX=/opt/xilinx/SDx/2019.1
-export XILINX_VIVADO=/opt/xilinx/Vivado/2019.1
-PATH=$PATH:/opt/xilinx/xrt/bin:$XILINX_SDX/bin:$XILINX_VIVADO/bin
-# This is the platform of the shell on the FPGA board
-export XILINX_PLATFORM=xilinx_u200_xdma_201830_1
```

Comment:
I also worked a improving documentation in this file.
very weirdly at least one of the test doesn't seem to terminate 
when XILINX_SDX isn't set so we still need it.
also for at least hw_emu LIBRARY_PATH needs to be set.

Reply:
OK, keeping it and I will investigate later.